catkin_virtualenv/cmake/templates/program.devel.in

```diff
@@ -1,2 +1,9 @@
-#!/usr/bin/env bash
-${${PROJECT_NAME}_VENV_DEVEL_DIR}/bin/python ${program_path} "$@"
+#!${${PROJECT_NAME}_VENV_DEVEL_DIR}/bin/python
```

Comment:
Nice. I wonder, did you test this with just this change (no `setproctitle`)? I was under the impression this might be enough but have never tried it.

Reply:
The interpreter name will still end up as part of sys.argv[0], so yeah, setproctitle (or similar) is still needed.

Reply:
Rebased but breaks tests - in CI this shebang ends up longer than the 128 character limit due to nested paths